### DISCUSSION TITLE
Redshift support

### DISCUSSION THREAD

**Discussion by lex970** [sequelize/sequelize#16704] [Help & Questions]
Hi there!

I want to ask about AWS Redshift support. I'm using sequelize 6.33.0, Node js 18.18.0. When I connect to the Redshift, I get a deprecation warning: 'DeprecationWarning: This database engine version is not supported, please update your database server.'
Redshift uses PostgreSQL 8.0.2 version. And when I added "databaseVersion: '8.0.2'" to sequelize config the DeprecationWarning went away. But sequelize version 6 documentation says that sequelize supports PostgreSQL Database engine from version 9.5. Basically, I want to ask you, can I work with sequelize and redshift without problem and and without unexpected behavior?

![Screenshot 2023-10-23 at 10 51 20](https://github.com/sequelize/sequelize/assets/112003750/cb9fa66e-88f9-4ab5-b398-2d27f049eb0d)

**Comment by WikiRik** [comment]
> Basically, I want to ask you, can I work with sequelize and redshift without problem and and without unexpected behavior?

No, we don't officially support Redshift. Since it uses postgres most things should work but support is up to the community and things can break in minor versions since we do not run our test suite on Redshift.

**Reply by lex970** [reply]
Thank you for your help!

**Reply by meyerovb** [reply]
Then why do you have Redshift in the main Readme?
